protobufs/crypto/wsts/wsts.proto

```diff
@@ -28,6 +28,22 @@ message SignerDkgPublicShares {
   crypto.Point kex_public_key = 4;
 }
 
+// DKG public shares message from a signer to all signers and coordinator.
+// This type is used by the signers for reading polynomial commitments from
+// the database. For sBTC, this vector should always have a length of 1.
+// This maps to this type:
+// <https://github.com/Trust-Machines/wsts/blob/2d6cb87218bb8dd9ed0519356afe57a0b9a697cb/src/net.rs#L137-L146>
+message DkgSignerCommitment {
+  // List of (signer_id, commitment)
```

Comment:
If a signer sends more than one of these, it's indexed on `party_id` not `signer_id`:
```suggestion
  // List of (party_id, commitment)
```

Reply:
Although calling it the `party_id` is more correct from a WSTS perspective, sBTC only uses the WSTS v2 system, so being more specific and calling it the `signer_id` seems to be a bit better.